src/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
don't force ports using liveserver pls,, it does a bunch of stuff in backend to find optimal port and some services run on these things just get rid of file

Reply:
my gitignore is not working. sad